spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/MessageGroupQueueTests.java

```diff
@@ -44,7 +43,7 @@
  */
 public class MessageGroupQueueTests {
 
-	@ClassRule
+//	@ClassRule
```

Comment:
?? If it's no longer long, then just remove the rule - or is this just left over from your testing?